spec/support/shared_examples/successfull_request_array_response.rb

```diff
@@ -0,0 +1,14 @@
+shared_examples 'successfull request array response' do
+  it 'responds with the expected keys and count' do
+    expect(response_body).to be_an(Array)
+    expect(response_body.size).to eq(expected.size)
+
+    response_body.each do |item|
+      expect(item.keys).to match_array(expected_keys)
+    end
```

Comment:
Buenisimo! También es válido, y lo vas a ver mucho, testear solo uno de los items:
```suggestion
      expect(response_body.first.keys).to match_array(expected_keys)
```